docs/DeveloperGuide.md

```diff
@@ -344,58 +359,56 @@ Priorities: High (must have) - `* * *`, Medium (nice to have) - `* *`, Low (unli
 
       Use case resumes at step 3.
```

Comment:
Also, not sure if this is intended, but the **Use case resumes at step 3.** is highly indented even when compared to the **Use case ends** above. <br>
![image](https://user-images.githubusercontent.com/95712150/193439519-fd413d44-7422-4686-8b8c-bd7e1ad8a95f.png)

Reply:
I think this appears quite a lot in this section.